os/pm/Kconfig

```diff
@@ -42,6 +43,63 @@ config PM_METRICS_DURATION
 
 endif
 
+comment "PM Options"
+
+config PM_NSTATE
+	int "Number of SoC sleep states"
+	default 0
+	---help---
+		Number of sleep states provided by the SoC Power module
+
+if PM_NSTATE > 0
```

Comment:
It doesn't work. please change it to `PM_NSTATE != 0`
As per Kconfig doc: 
```
<expr> ::= <symbol>                             (1)
           <symbol> '=' <symbol>                (2)
           <symbol> '!=' <symbol>               (3)
           '(' <expr> ')'                       (4)
           '!' <expr>                           (5)
           <expr> '&&' <expr>                   (6)
           <expr> '||' <expr>                   (7)
```
Non-Constant symbols are not part of expressions (Non-constant symbols are defined with the 'config' statement.)

Reply:
Have tested ..it works, but will change it to documented version